### PR TITLE
Travis CI: Add Python 3.8 production release

### PR DESCRIPTION
**Pull request by cclauss**
Also drop the EOL Python 3.4

### CODE REVIEW

**Comment by stefankoegl** [comment]
Thanks! I'll have a look

File: .travis.yml [hunk 1]
```diff
@@ -2,27 +2,25 @@ dist: xenial
...
 - travis_retry pip install coveralls
 script:
 - coverage run --source=jsonpointer tests.py
 after_script:
 - coveralls
-sudo: false
-addons:
-  apt:
-    packages:
-    - pandoc
```
Comment (stefankoegl) [inline .travis.yml]: Any particular reason why you moved that?, and removed the `sudo: false` ? I'd prefer to keep the diffs clean of unrelated changes.

Comment (cclauss) [reply .travis.yml]: Travis CI have completely deprecated the __sudo:__ tag.  The __sudo__ command is _always_ available in Travis and there is no longer anything that you can do to turn it off.

The move keeps things in chronological order because the apt tasks happen before the install tasks in the Travis CI logs.

**Comment by stefankoegl** [comment]
Thanks!

**Comment by stefankoegl** [comment]
Waiting with release for stefankoegl/python-json-pointer#39
